src/swapd/syncer_client.rs

```diff
@@ -294,11 +264,10 @@ impl SyncerState {
         self.tasks.tasks.insert(id, task.clone());
         task
     }
-    pub fn sweep_btc(&mut self, sweep_bitcoin_address: SweepBitcoinAddress, retry: bool) -> Task {
+    pub fn sweep(&mut self, addendum: SweepAddressAddendum, retry: bool) -> Task {
         let id = self.tasks.new_taskid();
         self.tasks.sweeping_addr = Some(id);
         let lifetime = self.task_lifetime(Blockchain::Bitcoin);
```

Comment:
this still needs to match on `SweepAddressAddendum::{Monero, Bitcoin}` variant

Reply:
I reworked it into two separate functions again.

Reply:
ack, I'll follow up with that refactor in another PR